### PR TITLE
To-do list webpack

### PR DESCRIPTION
**Pull request by Nejjari001**
The "To-do list" webpack project simply lists the things you need to do and allows you to mark them as complete.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+node_modules/
+.vscode/
+package-lock.json
+package.json
```
Comment (omar-muhamad) [inline .gitignore]: - [ ] Kindly remove `package.json` file from `.gitignore` and push it to github, So the reviewer or anyone can install the required dependencies (libraries) to run the app properly.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import retrieveFromStorage from './modules/retrieveFromStorage.js';
+import displayTasks, { footer } from './modules/displayTasks.js';
+import addTask from './modules/addTask.js';
+
+const totalTasks = JSON.parse(retrieveFromStorage('todo')) || [];
+const tasksListDisplay = document.querySelector('.display');
+
+if (totalTasks.length === 0) {
+  tasksListDisplay.innerHTML = '<hr/><p>No tasks available now!</p>';
+}
```
Comment (omar-muhamad) [inline src/index.js]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Create%20an%20index.js%20file%20and%20set%20an%20array%20of%20some%20simple%20to%20do%20tasks%20(array%20of%20objects).%20Each%20task%20object%20should%20contain%20three%20keys), Please add an array of tasks every task in an object that has 3 keys and values as described and render them on page load as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=On%20page%20load%20render%20the%20dynamically%20created%20list%20of%20tasks%20in%20the%20dedicated%20placeholder.%20The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.),

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Nejjari001,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please edit all README file sections with your project and personal data to make it more professional because now you only using the MIcroverse template without any editing.


## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Nejjari001 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - No linter errors. ✔️ 
 - Organized code. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**